modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm

```diff
@@ -14,4 +14,5 @@
 	)
 	default_pins = list(
 		/obj/item/mod/module/jetpack,
+		/obj/item/mod/module/armor_booster/nanotrasen/blueshield, //Bubberstation Addition - NT MOD Armour Boosters
```

Comment:
Can be made modular with
```
/obj/item/mod/control/pre_equipped/blueshield/Initialize()
	default_pins += /obj/item/mod/module/armor_booster/nanotrasen/blueshield
	. = ..()
```

Whilst you're at it, you can do the same to the rest of the bubber edits in this file which I appear to have missed modularizing the codebase

Reply:
I've done some of it, some I just let the override do its thing, and tried to clean up my code a bit to just have more of these Initialize procs instead of overriding to clean the code a bit better, let me know if there's any other way to improve it